docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[embedding]]
+== Embedding {kib}
+
+You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in *Share > Embed code* menu in *Dashboard* or *Visualize*.
+
+NOTE: Embedding of any other part of {kib} is also generally possible, but you may need to craft the proper HTML code manually.
+
+[role="screenshot"]
+image::images/embed-kibana.png[Generate an HTML snippet to embed {kib}, align=center]
+
+[float]
+[[embedding-security]]
+=== Configure security
+
+Embedding through iframes, if used properly, does not directly pose a security risk, but still requires a careful consideration.
```

Comment:
Is it worth explaining what some of these considerations are?

Reply:
I'll chime in here -- I think we should reword this. Nothing is zero-risk, and this statement could be misinterpreted. How about this?

```suggestion
Embedding content through iframes requires careful consideration to minimize security risks. By default, modern web browsers enforce the
https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy[same-origin policy] to restrict the behavior of framed pages. When
{stack-security-features} are enabled on your cluster, you must relax this constraint for cookies as described below for {kib} to function
in an iframe. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe[iframe] and
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite[SameSite] documentation for more information.
```

Note that we have a few links to MDN docs here, which there is precedent for in some other docs.

Reply:
> Note that we have a few links to MDN docs here, which there is precedent for in some other docs.

MDN is the well-known and trusted source of such information, I assume it should be fine. @gchaps do you have any concerns?